### PR TITLE
Node 16

### PR DESCRIPTION
**Pull request by sh0ji**
This is entirely a development dependency & maintenance update.

- Removes support for yarn, preferring npm.
- Various dependency updates in our documentation (Docusaurus + Storybook) to enable support for Node 16. This also removes support for Node 12 in our website, though not in the design system packages itself.
- CI and repo updates to use npm instead of yarn.
- Add Node 16 to CI testing.

In other words, developers working on the design system will need Node 14+, but developers using the design system packages can continue using Node 12+.

### CODE REVIEW

**Review by wguevara-norton — APPROVED** [review]
Thanks for adding npm support 🤓

**Comment by sh0ji** [comment]
That non-deterministic `LiveRegion` issue is quite annoying...I'm going to merge but will probably look into it soon.
